packages/k8s.contrib.crd/tests/ModuleGenerator.pkl

```diff
@@ -20,7 +20,8 @@ import "package://pkg.pkl-lang.org/pkl-k8s/k8s@1.0.1#/api/core/v1/EnvVar.pkl"
 import "package://pkg.pkl-lang.org/pkl-k8s/k8s@1.0.1#/api/networking/v1/NetworkPolicy.pkl"
 
 local generator = (generate) {
-  source = "file:crds.yaml"
```

Comment:
`read("file:crds.yaml")` isn't recommended, and actually doesn't have well defined behavior right now. It's best to do `read("crds.yaml")` to read relative file paths.